ogr/services/pagure/pull_request.py

```diff
@@ -191,18 +192,43 @@ def get(project: "ogr_pagure.PagureProject", pr_id: int) -> "PullRequest":
         return PagurePullRequest(raw_pr, project)
 
     @staticmethod
-    def get_files_diff(project: "ogr_pagure.PagureProject", pr_id: int) -> dict:
-        try:
-            return project._call_project_api(
-                "pull-request",
-                str(pr_id),
-                "diffstats",
-                method="GET",
-            )
-        except PagureAPIException as ex:
-            if "No statistics" in ex.pagure_error:
-                return {}
-            raise ex
+    def get_files_diff(
+        project: "ogr_pagure.PagureProject",
+        pr_id: int,
+        retries=3,
```

Comment:
I would set retries to 0 by default so that if called without optional arguments it behaves like a direct API call.

```suggestion
        retries=0,
```